src/System.Security.Cryptography.Pkcs/src/Resources/Strings.resx

```diff
@@ -244,6 +244,21 @@
   <data name="Cryptography_Pkcs_PssParametersSaltMismatch" xml:space="preserve">
     <value>PSS salt size {0} is not supported by this platform with hash algorithm {1}.</value>
   </data>
+  <data name="Cryptography_TimestampReq_BadNonce" xml:space="preserve">
+    <value>The response from the timestamping server did not match the request nonce.</value>
+  </data>
+  <data name="Cryptography_TimestampReq_BadResponse" xml:space="preserve">
+    <value>The response from the timestamping server was not understood.</value>
+  </data>
+  <data name="Cryptography_TimestampReq_Failure" xml:space="preserve">
+    <value>The timestamping server did not grant the request. The request status is '{0}' with failure info '{1}'.</value>
+  </data>
+  <data name="Cryptography_TimestampReq_NoCertFound" xml:space="preserve">
+    <value>The timestamping request requested the TSA certificate in the response, but it was not found.</value>
+  </data>
+  <data name="Cryptography_TimestampReq_UnexpectedCertFound" xml:space="preserve">
+    <value>The timestamping request requested the TSA certificate not be included in the response, but certificates were present.</value>
```

Comment:
Same thing here - should "request requested" actually be "request required"

Reply:
I can go either way.  Technically the request merely requested it.  The RFC then says that the response should only be accepted if this request was fulfilled.

Reply:
"Either way" + one more nudge => messages changed.